jade/applications/wsgi.py

```diff
@@ -0,0 +1,52 @@
+import docopt_subcommands as dsc
+from jade.wsgi import server
+
+from .util import build_config, configure_logging
+
+
+@dsc.command()
+def server_command(args):
+    """
+    usage:
+        {program} server (-h | --help)
```

Comment:
@halfak What do you think about this?  I'm torn between consistency across our tools ("applications.wsgi") and having a more human-readable command name.  The user doesn't care that this is an "application" nor that it uses "wsgi", only that it serves REST stuff...